app/assets/stylesheets/user-styles.css

```diff
@@ -28,10 +28,10 @@ img.right {
   margin: -8px; padding: 8px;
 }
 
-#sidebar .sidebar-title {
+#sidebar h3{
```

Comment:
I only noticed the h3 elements that needed to be changed in the #sidebar id. After changing this no other h3's on the page were effected.